pkg/pipelinerunmetrics/metrics.go

```diff
@@ -63,8 +63,6 @@ var (
 const (
 	// ReasonCancelled indicates that a PipelineRun was cancelled.
 	ReasonCancelled = "Cancelled"
-	// ReasonCancelledDeprecated Deprecated: "PipelineRunCancelled" indicates that a PipelineRun was cancelled.
-	ReasonCancelledDeprecated = "PipelineRunCancelled"
 )
```

Comment:
@vdemeester @jerop @imjasonh shall I revert this removal of deprecated reason or is this fine?

Reply:
In my opinion, we should keep `PipelineRunSpecStatusCancelledDeprecated` (and thus from user's perspective `PipelineRunCancelled`) for this release, allowing to use `Cancelled` as well as `PipelineRunCancelled` for a time being.
And we should add a deprecation notice on `PipelineRunSpecStatusCancelledDeprecated` (and thus from the user's perspective `PipelineRunCancelled`) that it will be removed in X releases (3 at least)

Reply:
for the deprecated reason, I think we should keep it as long as we keep the deprecated field value as well.